<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentTransactions</class>
 <widget class="QWidget" name="PaymentTransactions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>560</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #23272A;</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>820</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 900 18pt &quot;Segoe UI&quot;; color: white;</string>
   </property>
   <property name="text">
    <string>Payment Transactions</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>300</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QLineEdit {
       background-color: #1E1F22;
       color: white;
       border-radius: 8px;
       padding-left: 10px;
     }
     QLineEdit:focus {
       border: 1px solid #5865F2;
     }
    </string>
   </property>
   <property name="placeholderText">
    <string>Search transactions...</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="transactionsTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>820</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QTableWidget {
       background-color: #1E1F22;
       color: white;
       gridline-color: #5865F2;
       border-radius: 10px;
     }
     QHeaderView::section {
       background-color: #5865F2;
       color: white;
       font-weight: bold;
       border: none;
     }
    </string>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <column>
    <property name="text">
     <string>Transaction ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>User</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Reservation</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Amount</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="filterComboBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QComboBox {
       background-color: #1E1F22;
       color: white;
       border-radius: 8px;
       padding-left: 10px;
     }
     QComboBox:hover {
       border: 1px solid #5865F2;
     }
    </string>
   </property>
   <item>
    <property name="text">
     <string>All Transactions</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Today</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>This Week</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>This Month</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>30</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QPushButton {
       font: 700 10pt &quot;Segoe UI&quot;;
       color: white;
       background-color: #5865F2;
       border-radius: 8px;
     }
     QPushButton:hover {
       background-color: #4752C4;
     }
    </string>
   </property>
   <property name="text">
    <string>Export to CSV</string>
   </property>
  </widget>
  <widget class="QFrame" name="statsFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>820</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QFrame {
       background-color: #1E1F22;
       border-radius: 10px;
     }
    </string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="totalLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;; color: white;</string>
    </property>
    <property name="text">
     <string>Total Transactions: 125</string>
    </property>
   </widget>
   <widget class="QLabel" name="revenueLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>15</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;; color: #57F287;</string>
    </property>
    <property name="text">
     <string>Total Revenue: $12,500</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="exportButton_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>80</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QPushButton {
       font: 700 10pt &quot;Segoe UI&quot;;
       color: white;
       background-color: rgb(255, 0, 0);
       border-radius: 8px;
     }
     QPushButton:hover {
       background-color: rgb(170, 0, 0);
     }
    </string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
